tfjs-backend-webgpu/src/shader_preprocessor_wgsl.ts

```diff
@@ -246,6 +269,21 @@ const SAMPLING_SNIPPETS = `
     return u32(dot(vec4<f32>(coords), vec4<f32>(
         f32(shape.y) * f32(shape.z) * f32(shape.w), f32(shape.z) * f32(shape.w), f32(shape.w), 1.0)));
   }
+
+  // Only used when the y/z dimension of workgroup size is 1.
+  fn getGlobalIndex(globalId : vec3<u32>, localId : vec3<u32>, workGroupSize : vec3<u32>) -> u32 {
+    let workGroupID = (globalId - localId)/vec3<u32>(
```

Comment:
nit: Please move `let workGroupID = ` under the `if` block.

Reply:
Done